bindings/nodejs/lib/gpio.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2015 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ var soletta = require( 'bindings' )( 'soletta' ),
+    _ = require( 'lodash' );
+
+var edges = {
+    "none": soletta.sol_gpio_edge.SOL_GPIO_EDGE_NONE,
+    "rising": soletta.sol_gpio_edge.SOL_GPIO_EDGE_RISING,
+    "falling": soletta.sol_gpio_edge.SOL_GPIO_EDGE_FALLING,
+    "any": soletta.sol_gpio_edge.SOL_GPIO_EDGE_BOTH,
+}
+
+function pin_read_cb( pin, value ) {
+    _pins[ pin ].dispatchEvent( "change", {
+                        type: "change",
+                        value: value
+    } );
+}
+
+var _pins = new Array();
+
+exports.open = function( init ) {
+    return new Promise( function( fulfill, reject ) {
+        var pin = init.pin;
+        var dir = 0;
+        var drive_mode = 0;
+        var config = null;
+
+        if ( init.pullup )
+            drive_mode = soletta.sol_gpio_drive.SOL_GPIO_DRIVE_PULL_UP;
+        else
+            drive_mode = soletta.sol_gpio_drive.SOL_GPIO_DRIVE_PULL_DOWN;
```

Comment:
It's not a Boolean but an enum with 3 states, I'm not sure but there might be a reason, see http://solettaproject.github.io/docs/c-api/group__GPIO.html#ga9b8be46f5c047916a5cb7ecc51cf9ec6

@bdilly @ibriano or @mbelluzzo can you confirm and either change to Boolean or improve the C documentation?